tests/src/Unit/AvPortalResourceTest.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\media_avportal\src\Unit;
+
+use Drupal\media_avportal\AvPortalResource;
+use Drupal\Tests\UnitTestCase;
+
+/**
+ * Tests the AvPortalResource class.
+ */
+class AvPortalResourceTest extends UnitTestCase {
+
+  /**
+   * Tests getThumbnailUrl() method.
```

Comment:
```suggestion
   * Tests the testGetPhotoThumbnailUrl() method.
```
As I wrote in my previous review, we are testing the `getPhotoThumbnailUrl()` method. The `getThumbnailUrl()` method covers also videos.

I noticed now that the `getPhotoThumbnailUrl()` method would be used when the type is `PHOTO` or `REPORTAGE`. Let's add a `@dataProvider` called `photoTypeDataProvider()` method that returns `PHOTO` and `REPORTAGE` and let's use it in this method to check that it would work correctly.